politeiawww/user/cockroachdb/cockroachdb.go

```diff
@@ -756,6 +766,20 @@ func (c *cockroachdb) Close() error {
 	return c.userDB.Close()
 }
 
+// Quiesce toggles userdb quiesce mode.
```

Comment:
Function definition comments should use the full 80 characters column width.  Other comments are up to you.